civic_sandbox/helpers.py

```diff
@@ -29,42 +28,46 @@ def sandbox_function(request, date_filter=dates['default_date_filter'], format=N
                 ## converts MultiPolygons to Polygons ##
                 if isinstance(geom, MultiPolygon):
                     coords.append(geom.convex_hull)
-                elif isinstance(geom, MultiLineString):
-                    coords.append(geom.union)
+
+                #TODO merge multilinestring to linestring(https://docs.djangoproject.com/en/2.0/ref/contrib/gis/geos/#django.contrib.gis.geos.MultiLineString.merged)
+                # elif isinstance(geom, MultiLineString):
+                #     merged = geom.merged
+                #     coords.append(merged)
+
                 else: 
                     coords.append(geom)
             
             multi = multi_geom_class(coords)
             limit_boundary = multi.convex_hull.json
-            
 
-            print('made mps')  
+            if settings.DEBUG: print('boundary calculation complete')
 
             # calculate date meta #
             
             min_date = None
             max_date = None
+            if dates['date_attribute'] is not None: 
+                min_date = min(dates['date_attribute'])
+                max_date = max(['date_attribute'])
```

Comment:
I think this is a typo?  should be `max(dates['date_attribute'])`

Reply:
also what kind of values are you throwing at `min` and `max`?  Strings?  Integers?  Datetime objects?

Reply:
yes! thanks for catching that. I thought I fixed that but I guess not. Yeah, date or date time objects